### PR TITLE
Use ES module Ember import for addon version

### PR DESCRIPTION
**Pull request by charlesfries**
- Fixes `Ember` global warning in `ember-source@4.0.0`

### CODE REVIEW

**Comment by esbanarango** [comment]
@charlesfries Could you check this failing test:

```
Unit | Initializer | set-version TestLoader Failures dummy/tests/unit/initializers/set-version-test: could not be loaded
```

**Comment by charlesfries** [comment]
I am unable to reproduce this for the life of me. `regeneratorRuntime is not defined` is the error, and apparently it comes from Babel, but I'm not sure why that would be throwing with the change I made.

**Comment by charlesfries** [comment]
@esbanarango Finally figured it out, should be fixed now. Mocha/Chai didn't like the `async/await` syntax

File: tests/unit/initializers/register-version-test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    this.application = startApp({ autoboot: false });
+  });
+
+  afterEach(function() {
+    destroyApp(this.application);
+  });
+
+  // TODO: Replace this with your real tests.
+  it('works', function() {
+    this.application.boot().then(() => {
```
Comment (charlesfries) [inline tests/unit/initializers/register-version-test.js:25]: Do you know if `andThen` is preferred here? Sorry I'm not too familiar with Chai

**Review by esbanarango — APPROVED** [review]

**Pull request merged**